Library/Homebrew/github_packages.rb

```diff
@@ -53,11 +53,7 @@ def upload_bottles(bottles_hash, keep_old:, dry_run:, warn_on_error:)
       which("skopeo", ENV["HOMEBREW_PATH"]),
       HOMEBREW_PREFIX/"bin/skopeo",
     ].compact.first
```

Comment:
I think this logic could be pulled in too to optionally allow using the one from the `PATH` and avoid using `Formula[]` when unneeded?